R/helpers_rules.R

```diff
@@ -6,14 +6,19 @@
 #' divergence as given by equation in (7) in the reference at
 #' https://doi.org/10.1002/sim.8450.
 #'
-#' @param prob (`numeric`)\cr probability of a DLT occurring.
-#' @param target (`numeric `)\cr target probability of a DLT.
+#' @param prob (`numeric`)\cr vector or matrix with probabilities of a DLT occurring.
+#' @param target (`number `)\cr single target probability of a DLT.
 #' @param asymmetry (`number`)\cr describes the rate of penalization
 #'   for overly toxic does, range 0 to 2.
 #'
 #' @export
 #' @examples
-#' h_info_theory_dist(c(0.5, 0.2), c(0.5, 0.1), 1.2)
+#' h_info_theory_dist(c(0.5, 0.2), 0.4, 1.2)
```

Comment:
Makes sense, I guess the example worked before too because of vector calculations

Reply:
Yap, but I think at the moment we need to to be more secure rather than some extra flexibility, never used :)